third_party/rust/idna/v0_2/crate/src/lib.rs

```diff
@@ -0,0 +1,178 @@
+// Copyright 2016 The rust-url developers.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use std::{error::Error as StdError, fmt};
+
+#[derive(Default)]
+pub struct Errors {
+    punycode: bool,
```

Comment:
these fields can be removed if unused. the fields in the original type are private anyway

Reply:
https://github.com/brave/brave-core/pull/27961